### ISSUE TITLE
[Feature] Improve keyboard navigation

### ISSUE THREAD

**Issue by SetTrend** [2fast-team/2fast#66]
## Describe the problem this feature would solve

As a programmer, I'm rather focussed to handle apps using the keyboard than using the mouse or other gesture device. That's why I'd like to suggest to improve accessibility of the 2fast app by adding more keyboard shortcuts. This would ease handling a lot.

## Describe the solution

1. When the app opens, the list of registered clients should receive focus from the beginning.

2. The list of registered clients should be scrollable using cursor keys, like <kbd>UP ARROW</kbd> / <kbd>DOWN ARROW</kbd> / <kbd>HOME</kbd> / <kbd>END</kbd> / <kbd>PAGE UP</kbd> / <kbd>PAGE DOWN</kbd>.

3. Hitting <kbd>ENTER</kbd> (or <kbd>RETURN</kbd>, resp.) should copy the corresponding list item's current value to the clipboard:

   ![Improve keyboard support](https://user-images.githubusercontent.com/9283914/235900435-47c43397-fe38-483d-87fd-92fb7e38b15d.gif)

4. Moreover, hitting any of the number keys (<kbd>1</kbd> to <kbd>0</kbd>) should copy the corresponding list item's current value to the clipboard:
![2fa - 2](https://user-images.githubusercontent.com/9283914/235894659-85fd3a8b-668f-476b-8690-2850d72213ee.png)

5. Finally, I suggest the clickable area for copying a list item's value to be extended to encompass all three StackPanel items:
   1. The current value
   2. The copy icon
   3. The countdown icon

   ![2fa - 1](https://user-images.githubusercontent.com/9283914/235900739-31ac9006-12fa-4e86-8cc5-29e852290aa4.png)

This will make it easier to copy the current item's value without giving it too much thought.

**Comment by jp-weber** [comment]
Thank you for the corresponding suggestion :)

The implementation would be done in several steps, as the implementation of copying the code with tab and enter was easy to implement at first. 
![2fa_tab_and_copy](https://github.com/2fast-team/2fast/assets/10946850/10a433a6-63b7-41f0-b3b0-a5b5beaa0095)

The start and order of tab navigation is something i need to look at, this will probably require an option in the settings to not change the operation by default for ease of use or for the controller.

**Comment by SetTrend** [comment]
Marvellous job! Thanks!

**Comment by SetTrend** [comment]
Today I updated to the latest version. I looks like none of the changes you described is available in the production version yet?

**Comment by jp-weber** [comment]
The change was implemented after the release of current version 1.2.6, this will definitely be included in the next maintenance update 1.2.7.

**Comment by SetTrend** [comment]
Hi, just wanted to ring a bell for this feature request ☺️

**Comment by jp-weber** [comment]
This turns out to be more difficult than expected with the tab input. The initial main shell brings the focus to the hamburger button every time, but I don't want to overwrite the design instead only set the focus in the code behind. Even a focus on the listview of the accounts only works correctly once a navigation has been executed again.

The extended input with numbers should work if a control command is included (CTRL or something else).

**Comment by SetTrend** [comment]
Thank you for your kind reply!

Hmm, this is an UWP app, isn't it? Would the following suggestion help?

https://learn.microsoft.com/en-us/answers/questions/566115/how-to-set-focus-on-a-primary-button-in-an-input-d#answers

**Comment by jp-weber** [comment]
Yes, it is exactly this function, but it does not work immediately after logging in, it only works after a navigation, e.g. to the settings.

But I have implemented point four from your suggestion in version 1.3.1, see the RC release https://github.com/2fast-team/2fast/releases/tag/v1.3.1-rc1. 

If the list of accounts has the focus, you can navigate to the account by holding down the menu button (ALT) and entering the corresponding number (even with more than 9 accounts) and releasing the menu button.
